README.md

```diff
@@ -0,0 +1,9 @@
+# ScreenObject
+
+This package offers a starting point on which to implement the [page object](https://martinfowler.com/bliki/PageObject.html) pattern for UI testing iOS applications.
+In the context of mobile apps, there are screen, not pages, so we refer to the pattern as _screen object_.
```

Comment:
GitHub and Markdown are cool in that a new line like this is not rendered as a line break. This allows us to keep the line length short (so no/less need for horizontal scrolling) without compromising look of the rendered text.

Related: [Semantic Linebreaks](https://sembr.org/).

Reply:
In case you don't know, Markdown supports 2 types of rendered breaks:
 - When you do a single, regular CRLF like you did here, in GitHub it is not rendered and the CRLF is ignored in the final rendered doc. This can depend on the renderer used though (so if you're looking at the Markdown in an editor like MacDown or VSCode, rendering of those might differ slightly)
 - When you do double CRLFs (aka leave an empty new line between 2 paragraphs) it is rendered as a paragraph break and actually creates separate paragraphs once rendered. This is pretty common behavior across all Markdown renderers implementations
 - If you want to do a line break but not a paragraph break, also known as a soft break, aka a simple HTML `<br />` and not `<p>`, aka go to the new line within the same paragraph, you need to end the previous line with 2 spaces before your CRLF. This is quite nice (I use this all the time in e.g. SwiftGen CHANGELOG to go to the newline in the same `<li>` bullet point), the only drawback being that some editors trim trailing whitespaces and mess this up…

Reply:
>  you need to end the previous line with 2 spaces before your CRLF. This is quite nice 

It is indeed. I didn't know about this.

> the only drawback being that some editors trim trailing whitespaces and mess this up…

Yep, my Vim setup does that, too. I'm sure there's a way to tell it to trim all trailing whitespaces _unless_ they are two spaces 🤔